### PR TITLE
Use resources from dbcg/connectathon

### PR DESCRIPTION
**Pull request by lukoktonos**

### CODE REVIEW

**Review by mgramigna — CHANGES_REQUESTED** [review]
Looks good. One change that I have would be to use the new `fhir<3/4>/bundles/...` path motif for STU3 as well. Now that we are on the newer version of the connectathon repo for this Makefile, we can basically use the same paths for STU3 and R4 targets with just swapping the path and the version of the bundle.

**Comment by lukoktonos** [comment]
@mgramigna That's a good idea.  I'm not sure if it's the right call to do that in this PR, as it might affect @tstrass 's work more than this already does.  Should we make a separate Jira task to convert those paths for all measures all at once after the Makefile refactor gets in?

**Comment by mgramigna** [comment]
Yeah separate Jira is fine. I brought it up to Tom yesterday if I could work on that since there's nothing else in the sprint so we can separate it. As long as this still works for both versions as is then I'm fine.

**Comment by lukoktonos** [comment]
@mgramigna @tstrass added https://jira.mitre.org/browse/TACOSTRAT-375 to capture task discussed above.

**Review by mgramigna — CHANGES_REQUESTED** [review]
Not working for STU3 now, the ValueSets path that we were using doesn't exists on newest commit.

* `connectathon/fhir3/resources/valuesets/valuesets-EXM130_FHIR3-7.2.000.json` should be `connectathon/fhir3/resources/valuesets/valuesets-EXM130_FHIR3-7.2.000-bundle.json`
* `-m measure-exm130-FHIR3` in the `calculate-patients` target should be `-m measure-EXM130_FHIR3-7.2.000`

File: EXM_130/Makefile [hunk 1]
```diff
@@ -19,7 +19,6 @@ info:
 connectathon:
 	$(info connectathon checks out a specific commit SHA in case filepaths are updated)
```
Comment (okeefm) [inline EXM_130/Makefile]: Might want to remove this line since we're getting rid of the SHA

**Review by okeefm — CHANGES_REQUESTED** [review]
One tiny piece of feedback

File: EXM_130/Makefile (hunk 1 above)
Comment (lukoktonos) [reply EXM_130/Makefile]: Good catch!

**Comment by lukoktonos** [comment]
Current comments should be addressed

**Review by mgramigna — APPROVED** [review]

**Review by okeefm — APPROVED** [review]
Changes look good. Approved.

**Pull request merged**